scripts/shared/lib/clusters_kind

```diff
@@ -203,6 +208,32 @@ function deploy_ovn_cni(){
     echo "OVN CNI deployed."
 }
 
+function deploy_load_balancer() {
+    local kind_ip start_ip end_ip
```

Comment:
If LB is deployed on 2 clusters , IIUC both clusters will use the same IPAM range x1.x2.x3.100 - x1.x2.x3.250

We should have separate range for each cluster

Reply:
Yes, thats why cluster CIDR is used in the IP range, though now that I think of it probably better to use the GN CIDR..

Reply:
I updated so that GN CIDR is used when GN enabled, but generally your concern has already been addressed.